packages/grafana-schema/src/common/table.cue

```diff
@@ -17,6 +17,15 @@ TableSortByFieldState: {
 	desc?:       bool
 } @cuetsy(kind="interface")
 
+// TODO docs
```

Comment:
If we want to mark this field as `maturity:experimental` we should add this docs :)

Reply:
Yeah, was just going to say that it'd be great if each field in this schema is documented.